recipes-kernel/linux-firmware-rpidistro/linux-firmware-rpidistro_git.bb

```diff
@@ -57,7 +57,7 @@ do_install() {
     # Replace outdated linux-firmware files with updated ones from
     # raspbian firmware-nonfree. Raspbian adds blobs and nvram
     # definitions that are also necessary so copy those too.
-    for fw in brcmfmac43430-sdio brcmfmac43455-sdio brcmfmac43456-sdio ; do
+    for fw in brcmfmac43430-sdio brcmfmac43436-sdio brcmfmac43455-sdio brcmfmac43456-sdio ; do
```

Comment:
I'm pretty sure that these new boards were supposed to work with `brcmfmac43436s`. Can you test that and package them as well? You will need a package update for this.